ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/stubs/FakeKafkaClientSupplier.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.tools.stubs;
+
+import java.util.Collections;
+import java.util.Map;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.MockAdminClient;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.MockConsumer;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.clients.producer.MockProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.Node;
+import org.apache.kafka.streams.KafkaClientSupplier;
+
+public class FakeKafkaClientSupplier implements KafkaClientSupplier {
```

Comment:
Would you mind renaming this to `StubKafkaClientSupplier` please?   

I know this is just a copy of an existing class, but I really don't like the `Fake` in the name.  It's a stub, so let's call it as such.

Same for the other `Fake` classes.

Reply:
Done